graph/src/blockchain/mod.rs

```diff
@@ -73,7 +75,7 @@ pub trait Blockchain: Debug + Sized + Send + Sync + 'static {
     type TriggerData: TriggerData + Ord;
 
     /// Decoded trigger ready to be processed by the mapping.
-    type MappingTrigger: MappingTrigger;
+    type MappingTrigger: MappingTrigger + Debug;
```

Comment:
Silly question: is this the best place to put the `Debug`? I just don't know really 😅

Reply:
It could also be a super trait on `MappingTrigger`. Unsure of what's best.